payment_method_integration_test.go

```diff
@@ -40,9 +56,52 @@ func TestPaymentMethod(t *testing.T) {
 		t.Errorf("Got paymentMethod token %#v, want a value", paymentMethod.GetToken())
 	}
 
+	if card, ok := paymentMethod.(*CreditCard); ok {
+		ba := card.BillingAddress
+		if ba.FirstName != addr.FirstName {
+			t.Errorf("Got paymentMethod billing adress first name %#v, want %#v", ba.FirstName, addr.FirstName)
+		}
+		if ba.LastName != addr.LastName {
+			t.Errorf("Got paymentMethod billing adress last name %#v, want %#v", ba.LastName, addr.LastName)
+		}
+		if ba.Company != addr.Company {
+			t.Errorf("Got paymentMethod billing adress company %#v, want %#v", ba.Company, addr.Company)
+		}
+		if ba.StreetAddress != addr.StreetAddress {
+			t.Errorf("Got paymentMethod billing adress street address %#v, want %#v", ba.StreetAddress, addr.StreetAddress)
+		}
+		if ba.ExtendedAddress != addr.ExtendedAddress {
+			t.Errorf("Got paymentMethod billing adress extended address %#v, want %#v", ba.ExtendedAddress, addr.ExtendedAddress)
+		}
+		if ba.Locality != addr.Locality {
+			t.Errorf("Got paymentMethod billing adress locality %#v, want %#v", ba.Locality, addr.Locality)
+		}
+		if ba.Region != addr.Region {
+			t.Errorf("Got paymentMethod billing adress region %#v, want %#v", ba.Region, addr.Region)
+		}
+		if ba.PostalCode != addr.PostalCode {
+			t.Errorf("Got paymentMethod billing adress postal code %#v, want %#v", ba.PostalCode, addr.PostalCode)
+		}
+		if ba.CountryCodeAlpha2 != addr.CountryCodeAlpha2 {
+			t.Errorf("Got paymentMethod billing adress country alpha2 %#v, want %#v", ba.CountryCodeAlpha2, addr.CountryCodeAlpha2)
+		}
+		if ba.CountryCodeAlpha3 != addr.CountryCodeAlpha3 {
+			t.Errorf("Got paymentMethod billing adress country alpha3 %#v, want %#v", ba.CountryCodeAlpha3, addr.CountryCodeAlpha3)
+		}
+		if ba.CountryCodeNumeric != addr.CountryCodeNumeric {
+			t.Errorf("Got paymentMethod billing adress country numeric %#v, want %#v", ba.CountryCodeNumeric, addr.CountryCodeNumeric)
+		}
+		if ba.CountryName != addr.CountryName {
+			t.Errorf("Got paymentMethod billing adress country name %#v, want %#v", ba.CountryName, addr.CountryName)
+		}
+	} else {
+		t.Error("paymentMethod should have been a credit card")
+	}
+
 	// Update using different credit card
 	rand.Seed(time.Now().UTC().UnixNano())
 	token := fmt.Sprintf("btgo_test_token_%d", rand.Int()+1)
+	t.Fatal(token)
```

Comment:
Why always fail here?

Reply:
Was added during testing locally for this file. Will remove shortly.